geode-core/src/main/java/org/apache/geode/management/ManagementService.java

```diff
@@ -216,6 +216,11 @@ public abstract DistributedLockServiceMXBean getDistributedLockServiceMXBean(
    */
   public abstract Set<ObjectName> queryMBeanNames(DistributedMember member);
 
+  /**
+   * Returns the ids of the async event queues on this member
+   */
+  public abstract Set<String> getAsyncEventQueueIds(DistributedMember member);
```

Comment:
For consistency with the rest of the methods here, this should probably just be `getAsyncEventQueueMBeanNames` and return `Set<ObjectName>`.